src/OpenLoco/CompanyManager.cpp

```diff
@@ -195,8 +195,13 @@ namespace OpenLoco::CompanyManager
             auto company = get(id);
             if (company != nullptr && !isPlayerCompany(id) && !company->empty())
             {
-                setUpdatingCompanyId(id);
-                company->aiThink();
+                // Only the host should update AI, AI will run game commands
```

Comment:
Why? Its all deterministic so may as well make the ai do there thing locally.

Reply:
This could lead to a different game command order from the server. If a client and AI execute commands on the same tick, what comes first? I haven't really looked into it, but I just wanted to simplify things at the moment. Getting working AI in muliplayer is far in the future anyway.